api/libproject/devstart.rb

```diff
@@ -1058,7 +1058,7 @@ def circle_build_cdr_indices(cmd_name, args)
   op.add_option(
     "--branch [--branch]",
     ->(opts, v) { opts.branch = v},
-    "Branch. Optional - Default is master."
+    "Branch. Optional - Default is main."
```

Comment:
interesting - the default was already updated here

Reply:
I believe this one is referring to workbench-devops, which I moved over a while back (and apparently missed this doc update)